### DISCUSSION TITLE
Jsoncons convenience macros for rapidjson

### DISCUSSION THREAD

**Discussion by Dolers** [danielaparker/jsoncons#478] [Show and tell]
I've been inspired by the convenience macros in jsoncons, so I've adopted it to work with RapidJSON:
[https://github.com/Dolers/rapidjson_macros](https://github.com/Dolers/rapidjson_macros)

I hope I won't be considered presumptuous, but I'd like to leave the link in the readme to jsoncons' convenienece macros doc as it roots from it and also it explains the functionality perfectly.

Cheers!

**Comment by danielaparker** [comment]
Not at all, feel free to link.
